### PR TITLE
fix: evm token failing to migrate

### PR DESCRIPTION
**Pull request by jhasuraj01**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-7016 On cysync 2 new tokens are not getting synced with old database](https://app.clickup.com/t/9002019994/PRF-7016)

File: packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        const dataToUpdate: Partial<IAccount> = {};
+        (dataToUpdate.extraData as IEvmAccount['extraData']) = {
+          ...(account.extraData ?? {}),
+          lastContractTransactionBlockHeight: undefined,
+        };
+        await db.account.update({ __id: account.__id }, dataToUpdate);
+        logger.info('Contract Transaction Block Height: Reset Successful', {
+          assetId: account.assetId,
+          migration: migrationName,
+        });
```
Comment (irshadCypherock) [inline packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts]: Remove this log as success case is expected

File: .changeset/mighty-impalas-itch.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@cypherock/cysync-core-services': patch
+'@cypherock/cysync-desktop': patch
+---
+
+fix: evm token failing to migrate
```
Comment (irshadCypherock) [inline .changeset/mighty-impalas-itch.md]: Remove changelog as this issue is not related to the customer release

File: packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { IEvmAccount } from '@cypherock/coin-support-evm';
+import { coinFamiliesMap } from '@cypherock/coins';
+import { IAccount, IDatabase } from '@cypherock/db-interfaces';
+import logger from '../../../utils/logger';
```
Comment (irshadCypherock) [inline packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts:5]: Run linters and prettier

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: .changeset/mighty-impalas-itch.md (hunk 2 above)
Comment (jhasuraj01) [reply .changeset/mighty-impalas-itch.md]: done

File: packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts (hunk 1 above)
Comment (jhasuraj01) [reply packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts]: done

File: packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts (hunk 3 above)
Comment (jhasuraj01) [reply packages/core-services/src/db/migrations/helpers/resetContractTransactionBlockHeight.ts:5]: done

**Review by irshadCypherock — APPROVED** [review]
